doc/changes/unreleased.md

```diff
@@ -3,6 +3,7 @@
 ## ✨ Added
 
 * Added cookiecutter-template for creating new project
+* #74: Added security linter nox task
```

Comment:
This have been already released by now please update release notes of `0.15.0` which contains this change.
https://github.com/exasol/python-toolbox/blob/main/doc/changes/changes_0.15.0.md